app/helpers/permission_decorators.py

```diff
@@ -259,6 +259,8 @@ def decorated_function(*args, **kwargs):
             if user.is_registrar(event_id):
                 if '/attendees' in url:
                     return f(*args, **kwargs)
+                elif not request.is_xhr:
```

Comment:
Just want to make sure, I'm not redirecting any API requests. Redirect only when a registrar access any web page of the event other than attendees page.